components/style/properties/helpers.mako.rs

```diff
@@ -75,6 +74,7 @@
             }
             pub mod computed_value {
                 pub use super::single_value::computed_value as single_value;
+                pub use super::single_value::computed_value::T as SingleComputedValue;
```

Comment:
Yeah, I tried, it doesn't seem to work?

    error[E0432]: unresolved import `single_value::T`. Maybe a missing `extern crate single_value`?
       --> /media/Ubuntu/servo/target/geckolib/debug/build/style-8deb288d0c4a7300/out/properties.rs:315:25
        |
    315 |                 pub use single_value::T as SingleComputedValue;
        |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^